components/net/bluetooth_thread.rs

```diff
@@ -633,11 +633,22 @@ impl BluetoothManager {
             Some(a) => a,
             None => return drop(sender.send(Err(BluetoothError::Type(ADAPTER_ERROR.to_string())))),
         };
+        let device;
```

Comment:
I don't get why would you use this syntax :/
why not `let device = ...` in line 642?